content/academy/puppeteer_playwright/page/page_methods.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Page methods
+description: Understand that the Page object has many different methods to offer, and learn how to use two of them to capture a page's title and take a screenshot. 
+menuWeight: 3
+paths:
+    - puppeteer-playwright/page/page-methods
+---
+
+# [](#page-methods) Page methods
+
+Other than having methods for interacting with a page and waiting for events and elements, the **Page** object also supports many various methods for doing other things, such as [reloading](https://pptr.dev/#?product=Puppeteer&version=v13.7.0&show=api-pagereloadoptions), [screenshotting](https://playwright.dev/docs/api/class-page#page-screenshot), [changing headers](https://playwright.dev/docs/api/class-page#page-set-extra-http-headers), and collecting the [page's content](https://pptr.dev/#?product=Puppeteer&version=v13.7.0&show=api-pagecontent).
+
+Last lesson, we left off at a point where we are waiting for the page to navigate so that we can collect the page's title and take a screenshot of it. In this lesson, we'll be learning about the two methods we can use to easily achieve both of those things.
+
+## [](#grabbing-the-title) Grabbing the title
+
+There are two main page functions which will return general data:
+
+1. `page.content()` will return the entire HTML content of the page.
+2. `page.title()` will return the title of the current page found in the `<title>` tag.
+
+For our case, we'll utilize the `page.title()` function to grab the title and log it to the console:
+
+```JavaScript
+// Grab the title and set it to a variable
+const title = await page.title();
+
+// Log the title to the console
+console.log(title);
+```
+
+## [](#screenshotting) Screenshotting
+
+The `page.screenshot()` function will return a buffer which can be written to the filesystem as an image:
+
+```JavaScript
+import * as fs from 'fs/promises';
+// ...
+
+// Take the screenshot
+const screenshot = await page.screenshot();
+
+// Write the image to the filesystem
+await fs.writeFile('screenshot.png', screenshot);
+```
+
+> The image will by default be **.png**. To change the image to **.jpeg** type, set the (optional) `type` option to **jpeg**.
+
+## [](#final-code) Final code
+
+Here's our final code:
+
+```marked-tabs
+<marked-tab header="Playwright" lang="javascript">
+import puppeteer from 'puppeteer';
```

Comment:
The examples are mixed up. And in the previous lesson as well. Please fix it by swapping the code, not the titles. Playwright should always be the first example.